### ISSUE TITLE
How to do ensemble at testing with beam search?

### ISSUE THREAD

**Issue by homelifes** [sgrvinod/a-PyTorch-Tutorial-to-Image-Captioning#89]
Hi @sgrvinod, thanks for your code.
When using beam search, how de we perform ensemble testing (testing multiple models and average predictions across models). Should we add all the log softmax output and then divide them by the number of models?

**Comment by homelifes** [comment]
@sgrvinod im waiting for you reply
